firedrake/mesh.py

```diff
@@ -1996,6 +1996,8 @@ def SubDomainData(geometric_expr):
     The result can be attached as the subdomain_data field of a
     :class:`ufl.Measure`. For example:
 
+    .. code-block:: python
```

Comment:
Should be python3

Reply:
Changed `.. code-block:: python` -> `.. code-block:: python3` everywhere.